src/components/com_tjcertificate/script.tjcertificate.php

```diff
@@ -148,12 +148,6 @@ private function _installSubextensions($parent)
 						$installer = new JInstaller;
 						$result    = $installer->install($path);
 
-						$status->plugins[] = array(
-							'name' => 'plg_' . $plugin,
-							'group' => $folder,
-							'result' => $result
-						);
-
```

Comment:
is the code block not in use? @divyachaudhari

Reply:
Yes, Sir. this block is not used